test/e2e/framework/assertions.go

```diff
@@ -23,6 +23,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+const CustomForeverTestTimeout = 40 * time.Second
```

Comment:
(nit) feel free to ignore but I'd name the constant in an explicit manner for clarity...
```suggestion
const DefaultTestTimeout = 40 * time.Second
```

Reply:
fix in next pr